app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.elves.druid import Druid
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+
+
+def calculate_team_total_rating(
+        players: list[ElfRanger, Druid, DwarfWarrior, DwarfBlacksmith]
+) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elfs: list) -> None:
```

Comment:
Indicate that you expect to obtain instances of `Elf` class here. Extend type annotation for `feast_of_the_dwarves` as well